### ISSUE TITLE
Version 16.8.2

### ISSUE THREAD

**Issue by ntucker** [hot-loader/react-dom#4]
This is really annoying that this library is always behind important fixes.

**Comment by theKashey** [comment]
That's easy to solve:
- use RHL's webpack loader to land a patch. That's basically what this library is doing internally.
- become the team member and help keep it up to date.

**Comment by theKashey** [comment]
Released (that wasn't so easy)

**Comment by ntucker** [comment]
Hey, I'd love to contribute if I can but I don't think I understand this process. If it's documented somewhere and I missed it could you please point me to it?

Also if this is just webpack, do I need to use this if I am using webpack and use the plugin? I was under the impression that the mechanism for using webpack was simply setting up an alias to point to this repository.

**Comment by theKashey** [comment]
There is a `webpack-loader` in React-Hot-Loader, which could do some `babel-plugin` work, and injects some patches to the ReactDOM.
This is just an already patched ReactDOM - https://github.com/hot-loader/react-dom/blob/master/patch/build.js#L23 for the non-webpack devs or visa verca.

PS: There is no problem to release this package as soon as new React version got released, but I had to change the _patch_ to support hooks, and that took time.

PPS: There is no "process". Just jump in if you want.
